django/thunderstore/core/utils.py

```diff
@@ -1,3 +1,12 @@
+import os
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import transaction
+from sentry_sdk import capture_exception as capture_sentry_exception
```

Comment:
When we init the sentry SDK, we run our inputs only if a DSN is actually provided:
https://github.com/thunderstore-io/Thunderstore/blob/b5220fb002e08cd9e817233af843921810b35306/django/thunderstore/core/settings.py#L68-L70
Maybe we should move this import to if it's needed? This is different though as it is in a function but `sentry_sdk` isn't an optional dependency, so maybe we could move the import in `settings.py`.

Related: https://github.com/thunderstore-io/Thunderstore/pull/196#discussion_r574770573